common/include/common/blas_extension_state_counters.hpp

```diff
@@ -40,21 +40,31 @@ enum class ExtensionOP : int {
 };
 
 template <ExtensionOP op, typename scalar_t, typename index_t>
-inline typename std::enable_if<op == ExtensionOP::omatcopy ||
-                               op == ExtensionOP::imatcopy ||
-                               op == ExtensionOP::omatcopy2>::type
+inline typename std::enable_if<
+    op == ExtensionOP::omatcopy || op == ExtensionOP::imatcopy ||
+    op == ExtensionOP::omatcopy2 || op == ExtensionOP::omatcopy_batch ||
+    op == ExtensionOP::imatcopy_batch>::type
```

Comment:
We don't mention `imatcopy_batch` in the documentation, can we sync the documentation with the code to make sure all the relevant information is mentioned there?